src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,162 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int size;
+    private int threshold;
+    private Node<K, V>[] table;
+
+    static class Node<K, V> {
+        private int hash;
+        private K key;
+        private V value;
+        private Node<K, V> next;
+
+        public Node(int hash, K key, V value, Node<K, V> next) {
+            this.hash = hash;
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+
+        public final boolean equals(Object o) {
+            if (o == this) {
+                return true;
+            }
+            if (o.getClass().equals(Node.class)) {
+                Node e = (Node) o;
+                return e.key.equals(key) && e.value.equals(value);
+            }
+            return false;
+        }
```

Comment:
You don't need it here. Also, remember that we always override both equals and hashcode. Don't forget @Override annotation